tests/flows/test_flowstatus.py

```diff
@@ -13,12 +14,12 @@
 
 from siliconcompiler.tools.builtin import nop
 from siliconcompiler.tools.builtin import minimum
+from siliconcompiler import SiliconCompilerError
 
 
 @pytest.mark.eda
 @pytest.mark.quick
 @pytest.mark.parametrize('steplist', [
-    ['import', 'place'],
```

Comment:
why is this one removed?

Reply:
okay, can you add a unit test to ensure a forked run will be marked a failure?

Reply:
`test_long_branch` tests the same disjoint branches with failures and I changed the behavior there.
I think this would be duplicate.